nvtabular/ops/categorify.py

```diff
@@ -104,6 +105,18 @@ class Categorify(DFOperator):
         for multi-column groups.
     search_sorted : bool, default False.
         Set it True to apply searchsorted algorithm in encoding.
+    num_buckets : int, list of int, or dictionary:{column: num_hash_buckets}
+        Column-wise modulo to apply after hash function. Note that this
+        means that the corresponding value will be the categorical cardinality
+        of the transformed categorical feature. If given as an int, that value
+        will be used as the number of "hash buckets" for every feature. If
+        a list is provided, it must be of the same length as `columns` (which
```

Comment:
@rjzamora I actually agree on that. It would be confusing. I am fine removing `list` option (it was taken from HashBucket op) to avoid confusions/problems.